pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -97,6 +98,7 @@ def __init__(self, **kwargs):
 
     def do_build_flow(self, **kwargs):
         flow = linear_flow.Flow('grpc-codegen')
+        flow.add(io_tasks.PrepareDirsTask('PrepareDirsTask', inject=kwargs))
```

Comment:
style nit: probably better to keep the consistent style -- use `flow.add(io_task.PrepareDirsTask(...), protoc_tasks....)` rather than invoking flow.add twice.